### ISSUE TITLE
Improve test coverage with practical data sets

### ISSUE THREAD

**Issue by tbirdso** [KitwareMedical/ITKUltrasound#188]
Several ultrasound image filters rely on contrived data for CTest coverage. For example, `Spectra1DAveragingImageFilter` tests currently average over multiple copies of a single image, `spectra_pixel_image.mha`, which depicts a 3D image with a single 31-channel pixel. In this case limited test data failed to cover an issue with stepping over lines, leading to the patch in #187.

It would be useful to revisit our testing and potentially duplicate some image filter tests with a small variety of input data samples representing real-world cases.

**Comment by thewtex** [comment]
@dzenanz has some test data for spectral data.

**Comment by tbirdso** [comment]
To that point, we already host/include liver spectra samples for attenuation testing (see [data.kitware.com](https://data.kitware.com/#collection/57b5c9e58d777f126827f5a1/folder/6218eff24acac99f425cf6a3)). @dzenanz should I amend this issue to reflect adding `Spectra1DAveragingImageFilter` and `Spectra1DNormalizeImageFilter` tests on sample data? Are there any other tests you are aware of that should be expanded?

**Comment by dzenanz** [comment]
Yes, testing on that sample data would be good. Those two filters were recently added, I doubt more filters need extra testing.

But [Phantom RF data](https://data.kitware.com/#collection/57b5c9e58d777f126827f5a1/folder/610c449d2fa25629b9e6535c) could be used as start of chain of filters: `Spectra1DSupportWindowImageFilter`+`Spectra1DImageFilter` -> `Spectra1DAveragingImageFilter`. Then apply `Spectra1DNormalizeImageFilter` to liver data using output of `Spectra1DAveragingImageFilter` as reference for normalization.

**Comment by tbirdso** [comment]
@dzenanz I've added #189 as a sanity check applying averaging and normalization separately to practical data. I agree that the workflow you've suggested here would make a good integration test, especially in light of recent discussion around spectra image issues.
